dns/otcep.gov.sg.ts

```diff
@@ -0,0 +1,34 @@
+import { Record } from "@pulumi/aws/route53";
+import { CLOUDFRONT_HOSTED_ZONE_ID } from "../constants";
+
+export const createRecords = (zoneId: string): Record[] => {
+  const records = [
+    new Record("otcep.gov.sg A", {
+      name: "otcep-gov-sg",
+      type: "A",
+      zoneId: zoneId,
+      aliases: [
+        {
+          name: "d38bc0wjigwtgf.cloudfront.net",
+          zoneId: CLOUDFRONT_HOSTED_ZONE_ID,
+          evaluateTargetHealth: false,
+        },
+      ],
+    }),
+
+    new Record("otcep.gov.sg AAAA", {
+      name: "otcep-gov-sg",
+      type: "AAAA",
+      zoneId: zoneId,
+      aliases: [
+        {
+          name: "d38bc0wjigwtgf.cloudfront.net",
+          zoneId: CLOUDFRONT_HOSTED_ZONE_ID,
+          evaluateTargetHealth: false,
+        },
+      ],
+    }),
+  ];
+
+  return records;
```

Comment:
/superNit It's fine to have code/value duplication for a small 2-elements array. But even for 2 entries, the typical issues still apply that 1) if you need to make a change you need to change multiple things, and 2) it's not always immediately obvious if something is an exact duplicate of another, or if there are some subtle differences.

One cool aspect of using pulumi is that you _can_ structure your infra declaration to drop duplication with standard code refactoring techniques.

In this case, we could consider doing this:

```suggestion
  return ['A', 'AAAA'].map(type => new Record(
      `otcep.gov.sg ${type}`,
      {
        name: 'otcep-gov-sg',
        type,
        zoneId,
        aliases: [
          {
            name: "d38bc0wjigwtgf.cloudfront.net",
            zoneId: CLOUDFRONT_HOSTED_ZONE_ID,
            evaluateTargetHealth: false,
          },
        ],
      }
    )
  )
```

Arguably, it's not as readable as the duplicated code (? 🤔), so trade off, trade off, you decide. Feel free to ignore this /nit comment 😀